src/commands/mget.json

```diff
@@ -6,9 +6,16 @@
         "since": "1.0.0",
         "arity": -2,
         "function": "mgetCommand",
+        "history": [
+            [
+                "9.0.0",
+                "Supports cross-slot keys in cluster mode."
```

Comment:
I prefer to indicate more detail:  Supports cross-slot keys in same shard in cluster mode.